qbraid/runtime/rigetti/device.py

```diff
@@ -0,0 +1,61 @@
+from typing import Any, Dict
+
+
+from qbraid.runtime import (
+    QuantumDevice,
+    TargetProfile,
+    RuntimeOptions,
+)
+import pyquil
+import pyquil.api
+
+from qbraid.runtime.enums import DeviceStatus
+
+class RigettiDevice(QuantumDevice):
+    """
+    Wraps a single Rigetti QCS quantum processor or simulator.
+    """
+
+    def __init__(
+        self,
+        profile: TargetProfile,
+        qc: pyquil.api.QuantumComputer,
+    ):
+        """
+        profile: A TargetProfile object (constructed by RigettiProvider).
+        """
+        # Call base class initializer, passing the TargetProfile
+        super().__init__(profile=profile, scheme=None, options=RuntimeOptions())
```

Comment:
```suggestion
        super().__init__(profile=profile)
```

initialization of conversion scheme and runtime options done automatically in parent class